packages/libao/ao_private.h.patch

```diff
@@ -0,0 +1,11 @@
+--- ../cache/libao-1.2.2/include/ao/ao_private.h	2016-11-14 16:03:30.000000000 +0000
++++ ./include/ao/ao_private.h	2018-07-10 21:09:26.757188903 +0000
+@@ -51,7 +51,7 @@
+ /* --- Constants --- */
+
+ #ifndef AO_SYSTEM_CONFIG
+-#define AO_SYSTEM_CONFIG "/etc/libao.conf"
++#define AO_SYSTEM_CONFIG "/data/data/com.termux/files/user/etc/libao.conf"
```

Comment:
Please use `@TERMUX_PREFIX@` instead of `data/data/com.termux/files/usr`, it is sed'ed before the patch is applied in build-package.sh.